Formerly only needed with python-3.14.x but backports have broke
this with python-3.13.6 too.

https://bugs.gentoo.org/960115
https://bugs.gentoo.org/961230
https://code.videolan.org/videolan/libplacebo/-/commit/12509c0f1ee8
--- a/src/vulkan/utils_gen.py
+++ b/src/vulkan/utils_gen.py
@@ -203,5 +203,6 @@
         xmlfile = find_registry_xml(datadir)
 
-    registry = VkXML(ET.parse(xmlfile))
+    tree = ET.parse(xmlfile)
+    registry = VkXML(tree.getroot())
     with open(outfile, 'w') as f:
         f.write(TEMPLATE.render(
